index.php

```diff
@@ -0,0 +1 @@
+<?php header( 'Location: /index.html' ) ;  ?>
```

Comment:
If this file isn't being used for anything, you could remove it to have better organization